docs/conf/writing.md

```diff
@@ -220,6 +220,8 @@ clientSecret: someGithubAppClientSecret
 webhookSecret: someWebhookSecret
 privateKey: |
   -----BEGIN RSA PRIVATE KEY-----
-  SomeRsaPrivateKey
+  SomeRsaPrivateKeyForExampleOnly
   -----END RSA PRIVATE KEY-----
 ```
+
+**Warning: RSA private keys should not be hard coded**. Keep them in a secure storage solution like Vault, to ensure they are neither exposed nor misused.
```

Comment:
I would even go a step further and say that we recommend that this entire file should be a secret and stored as such. It's a little awkward with the RSA private keys to load the private key content from elsewhere and make sure that it's trimmed and has the right formatting etc.

It's much easier to store the entire file and mount that as a secret somewhere I think.

Reply:
I was able to get the private key in as an environment variable by converting to newline character `\n`. Just didn't feel like dealing with volume mounts

Reply:
Hi, yes indeed 🙂 That's what I meant, sorry if it wasn't clear enough. I try again !